### PR TITLE
e2e: Increase all ANR timeouts to 2m to ensure CI reliability.

### PR DESCRIPTION
**Pull request by marun**
## Why this should be merged

CI seems to be exceeding many ANR-related timeouts. Rather than bumping timeouts piecemeal, all ANR timeouts are set to the same constant of 2 minutes.

### CODE REVIEW

**Review by abi87 — APPROVED** [review]
Thanks for spotting this!

**Review by dhrubabasu — APPROVED** [review]

File: tests/e2e/e2e.go [hunk 1]
```diff
@@ -167,10 +167,7 @@ func (te *TestEnvironment) StartCluster() error {
 		}
 		tests.Outf("{{green}}successfully started network-runner: {{/}} %+v\n", resp.ClusterInfo.NodeNames)
 
-		// start is async, so wait some time for cluster health
-		time.Sleep(time.Minute)
```
Comment (StephenButtolph) [inline tests/e2e/e2e.go:171]: Was this sleep just never needed? Like a minute long sleep is _long_

Comment (felipemadero) [reply tests/e2e/e2e.go:171]: Not neede anymore

Comment (StephenButtolph) [reply tests/e2e/e2e.go:171]: The missing piece here is understanding that `Health` actually blocks until `Healthy`.

Comment (felipemadero) [reply tests/e2e/e2e.go:171]: yup that was very old ANR tech dep. with the first grpc server implementation, start (with blockchain creation -I know this is not the case but this is a copy probably-) broked posterior health call without sleeping some time.

**Pull request merged**